Dockerfile

```diff
@@ -0,0 +1,29 @@
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+FROM python:3.5-alpine
```

Comment:
Elsewhere we are using `FROM python:3.5`.

It might be good to standardise on some base everywhere in order to avoid surprises, see for example why Elastic moved away from Alpine https://www.elastic.co/blog/docker-base-centos7